src/content/tools/dart-format.md

```diff
@@ -66,21 +66,13 @@ so they can trigger another action in response to the exit code.
 $ dart format -o none --set-exit-if-changed bin/my_app.dart
 ```
 
-## Use trailing commas
+## Whitespace and commas
```

Comment:
@munificent I considered the points you made in the issue and thought this might be a good in between vs. getting rid of the sections all together. Do you think a brief summary about what the format generally changes, like I added here, is useful?

I might also add to this list that `format` will format the line length too (https://github.com/dart-lang/site-www/issues/6223) so lmk if that makes sense too.